attractions/utils/datetime-utils.js

```diff
@@ -206,14 +215,35 @@ export function datesLessEqual(date1, date2) {
   );
 }
 
+/**
+ * Checks if the date is included in the given array of dates or ranges.
+ * @param {Date} date The date object, which inclusion to check
+ * @param {Array<Date | { start?: Date; end?: Date; }>} dateRanges The set of dates
+ * @returns {boolean}
+ */
+export function dateIncluded(date, dateRanges) {
```

Comment:
I was thinking since this got refactored to its own function, its call should be decoupled from `getCalendar`, and instead be composed in the Calendar component.
At least that was before you used `day.disabled` in two props instead of one, so I'm no longer sure.

Reply:
Also, should its documentation be added to the utilities docs or do we only document ones directly exported from `utils/index`?

Reply:
I'd prefer we only document the direct exports, and consider the rest to be internal utilities